python_modules/dagster/dagster/_core/definitions/definitions_class.py

```diff
@@ -114,7 +113,48 @@ def created_repo():
 
 class Definitions:
     """
-    A set of definitions to be explicitly available and loadable by Dagster tools.
+    A set of definitions explicitly available and loadable by Dagster tools.
+
+    Parameters:
+        assets (Optional[Iterable[Union[AssetsDefinition, SourceAsset, CacheableAssetsDefinition]]]):
+            A list of assets for this repository. Assets can be created by annotating
```

Comment:
I feel like we should stay away from `repository` unless we're specifically referring to something like `a GitHub repository`. I've seen some folks in the Dagster Slack define `@repository` and then use `Definitions` alongside it, leading to unexpected results - I think this language could be confusing.

Reply:
Oh yeah, +1

Reply:
Thanks great catch